src/OpenTelemetry/Metrics/Aggregators/Aggregator.cs

```diff
@@ -47,5 +61,23 @@ public abstract class Aggregator<T>
         /// </summary>
         /// <returns><see cref="AggregationType"/>.</returns>
         public abstract AggregationType GetAggregationType();
+
+        /// <summary>
+        /// Determines the last end timestamp before the current interval's <see cref="startTime"/>.
```

Comment:
refering to private fields in public document is not ideal. Can it be rephrased without the mention of private fields?

Reply:
Maybe mention other public method like `GetLastEndTimestamp ` instead

Reply:
Done